src/presentation/ui/page/accounts.templ

```diff
@@ -0,0 +1,268 @@
+package page
+
+import (
+	"github.com/goinfinite/os/src/domain/entity"
+	componentForm "github.com/goinfinite/os/src/presentation/ui/component/form"
+	componentStructural "github.com/goinfinite/os/src/presentation/ui/component/structural"
+)
+
+script AccountsIndexLocalState() {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('accounts', () => ({
+			// Primary states
+			account: {},
+			resetPrimaryStates() {
+				this.account = {
+					id: '',
+					groupId: '',
+					username: '',
+					password: '',
+					apiKey: '',
+				}
+			},
+			init() {
+				this.resetPrimaryStates();
+			},
+
+			// Auxiliary states
+			get shouldDisableUpdatePasswordSubmitButton() {
```

Comment:
You like to suffer, gotta be it. Man, you should ditch those should disable buttons. Nobody like that and you spent precious dev time adding that.

Reply:
But it works... :(
Anyway, I've removed it.